### PR TITLE
Make use of consolidated check-resource common role

### PR DESCRIPTION
**Pull request by ramperher**

### CODE REVIEW

**Review by tonyskapunk — COMMENTED** [review]
This change LGTM, but I'm not sure what's the gain on the change of the role, I'll add more comments in the change of that role.

**Review by manurodriguez — APPROVED** [review]
LGMT

**Review by tkrishtop — APPROVED** [review]
LGTM thanks Ramon

File: testpmd/hooks/pre-run.yml
Comment (ramperher) [reply testpmd/hooks/pre-run.yml]: hehe, nice catch! I've removed it

**Comment by ramperher** [comment]
I will wait till having d-o-a and d-o-a-a changes merged before merging this, and also after confirming a new job passes.

**Pull request merged**
